src/constructors.md

```diff
@@ -45,15 +75,40 @@ constructor, but it's never implicitly invoked. You have to explicitly call
 where the implementation is just "copy the bits". Copy types *are* implicitly
 cloned whenever they're moved, but because of the definition of Copy this just
 means not treating the old copy as uninitialized -- a no-op.
+-->
 
+Assignment コンストラクタや Copy コンストラクタも同様に存在しません。
+なぜなら、ムーブセマンティクスは Rust における唯一のセマンティクスだからです。
+せいぜい `x = y` が単に y のビットを変数 x に移すくらいです。 Rust では C++ の
+コピー指向のセマンティクスを提供する、 2 つの機能があります。 `Copy` と `Clone` です。 Clone は Copy コンストラクタと
+同じようなものですが、暗黙に呼び出されることは一切ありません。クローンを生成したい要素に対して、
+明示的に `clone` を呼び出す必要があります。 Copy は Clone の特別なケースで、
+実装は単純に "ビットをコピーする" ことです。 Copy を実装する型は、
+ムーブが発生すると毎回クローンを生成*します*。しかし、 Copy の定義によって、
+これは、古いコピーを初期化されていないとは扱わない事を単に意味します。つまり no-op なのです。
```

Comment:
I do not really understand the original sentence. Probably it means "`Copy` does not drop `x` in `let y = x`" ?
If so, I would say
古いコピーを未初期化状態にはしない事を単に意味します。
though, this would not be a faithful translation.